src/components/SignInU/SignInU.js

```diff
@@ -0,0 +1,139 @@
+import React, {Component} from 'react';
+import './SignInU.css';
+import Input from '../../components/Input/Input';
+import {withRouter} from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { signin } from '../../store/reducers/signinSlice';
+
+class SignInU extends Component {    
```

Comment:
What 'U' means here? What's the difference with SignIn? Why you need duplicating components?

Reply:
SignIn removed. SignInU - new component.